main.tf

```diff
@@ -17,6 +17,13 @@ resource "azurerm_key_vault" "kv" {
   enabled_for_template_deployment = true
   soft_delete_retention_days      = 90
 
+  network_acls {
+    bypass         = "AzureServices"
+    default_action = var.network_acls_default_action # Default is "Deny"
```

Comment:
```suggestion
    default_action = var.network_acls_default_action # Default is "Allow" for compatibility
```

Reply:
Default is changed to "Allow" for compatibility